### ISSUE TITLE
Allow loading additional custom fonts without overriding system fonts.

### ISSUE THREAD

**Issue by ceyhunali** [AnemoneTeam/Anemone#163]
Anemone doesn't change Calendar Icon Font. 
Please add support.

**Comment by cj81499** [comment]
@iCeka Check here for information on theming the Calendar Icon https://github.com/AnemoneTeam/Anemone/wiki/Calendar-Icon

**Comment by ceyhunali** [comment]
@cj81499 sorry, I meant that I have added custom .ttf files to Example Anemone Font folder to use with Anemone. But it doesn't change Calendar date icon font type. About Anemone new font support feature.

**Comment by cj81499** [comment]
@iCeka Ah. I see. I don't know much about the font side. @coolstar and @mwoolweaver are most likely to be able to help with this.

**Comment by ceyhunali** [comment]
@cj81499 my bad, no problem, thank you anyway.

**Comment by gmawji** [comment]
Bumping this. I'd like to use custom fonts on the calendar also if possible.

**Comment by coolstar** [comment]
So I'm guessing rather than swapping fonts you'd like to load your own custom fonts that aren't provided in iOS without overriding any of iOS's system fonts?

This is something we can do.

**Comment by gmawji** [comment]
So here was my idea. I have fonts for my theme. And they change the system fonts but I want to pull a font to have my calendar date and day show a different font

**Comment by coolstar** [comment]
yeah so it sounds like you want to load an additional font from your theme, which would let you set it for the calendar icon via Info.plist

**Comment by gmawji** [comment]
Yes @coolstar

**Comment by smokepcpeveryday** [comment]
did this ever come to be anything?
